apps/meteor/app/apps/server/converters/rooms.js

```diff
@@ -172,9 +175,14 @@ export class AppRoomsConverter {
 					return undefined;
 				}
 
+				const { lastMessageTs } = v;
+
 				delete room.v;
 
-				return this.orch.getConverters().get('visitors').convertById(v._id);
+				return {
+					...(await this.orch.getConverters().get('visitors').convertById(v._id)),
+					...(lastMessageTs && { lastMessageTs }),
```

Comment:
I would really recommend a comment above this line clarifying why we're adding this field here. On a first glance, it looks like the filed should just be added to the `VisitorsConverter.convertById` method. Leaving it like this will generate confusion in the future

Reply:
Good idea! Added a comment here :heavy_check_mark: